src/app/fdctl/run/tiles/fd_net.c

```diff
@@ -218,6 +234,29 @@ during_frag( void * _ctx,
   fd_memcpy( ctx->frame, src, sz ); // TODO: Change xsk_aio interface to eliminate this copy
 }
 
+static void
+send_arp_probe( void * _ctx,
+                uint   dst_ip_addr,
+                uint   ifindex ) {
+  fd_net_ctx_t * ctx     = (fd_net_ctx_t *)_ctx;
+  uchar *        arp_buf = ctx->arp_buf;
+  ulong          arp_len = 0UL;
+
+  uint           src_ip_addr  = ctx->src_ip_addr;
+  uchar *        src_mac_addr = ctx->src_mac_addr;
+
+  /* prepare arp table */
+  int arp_table_rtn = fd_ip_update_arp_table( ctx->ip, dst_ip_addr, ifindex );
+  (void)arp_table_rtn;
```

Comment:
Can you add a metric for the result of this update call. And also, the error cases should be handled (don't send the probe).

Reply:
We want to send the probe anyway. It'll will be quite likely for fd_ip_update_arp_table to fail due to it already having been executed successfully, possibly in a different tile.

Reply:
I don't fully follow this. It could be NUD_UNREACHABLE in which case probing is pointless? I presume there are other cases too, if there's already a probe in flight and things like that? As well, we still want a metric for the result code here.

Reply:
There is no NUD_UNREACHABLE
ARP sends a broadcast, and hopes for a reply. The failure modes are "can't send a broadcast", and "timed out"
I'll add a metric

Reply:
It's NUD_NOARP, which does seem like a valid result.